vips/foreign.h

```diff
@@ -19,7 +19,18 @@ enum types {
 	BMP
 };
 
-int load_image_buffer(void *buf, size_t len, int imageType, VipsImage **out);
+typedef struct {
+    gboolean fail_set;
```

Comment:
Why did you decide to have separate variables vs just using pointers with null checks?

Reply:
I'm not sure if I can put together the correct construct with a pointer at the bridge of Go and C.